src/utils/prismImport.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable filenames/match-regex */
```

Comment:
Why did we disable `filenames` for this file?

Reply:
The rule didn’t like something about the prism core import :(

Reply:
It's complaining about `src/utils/prismImport.js` name of this file, not the `import`s within the code I believe.

Reply:
Actually, all the _prismjs_ imported file names abide by the lint rule we have for this repo 😛

Reply:
Oh, ok. Sorry, I’ll make a change